src/Stratis.Bitcoin.Features.PoA/Voting/PollsRepository.cs

```diff
@@ -81,6 +81,31 @@ private void SaveHighestPollId(DBreeze.Transactions.Transaction transaction)
             transaction.Insert<byte[], int>(TableName, RepositoryHighestIndexKey, this.highestPollId);
         }
 
+        /// <summary>Removes polls for the provided ids.</summary>
```

Comment:
The duplication between this and `RemovePolls` is unfortunate, but I suppose the use cases are slightly different

Reply:
Yeah there is subtle difference in the logic, in that we set/calculate the highest poll id differently.